.github/CODEOWNERS

```diff
@@ -1,8 +1 @@
-*       @g-k, @kkleemola, @pwnbus
-
-# the AMO team should review changes that impact Addon signing
-signer/xpi/	@willdurand
```

Comment:
If possible, I'd like to be pinged explicitly when there are changes related to add-ons.

Reply:
oh! Is there an add-ons reviewer team?

Reply:
Can you try https://github.com/orgs/mozilla-services/teams/addons-engineering please?

Reply:
@willdurand thank you for the team suggestion, I updated the PR to use that